### DISCUSSION TITLE
support to the kimi ai

### DISCUSSION THREAD

**Discussion by xiaodashuaige** [logancyang/obsidian-copilot#551] [Q&A]
the local llm are slow to response and there are some difficulties for me to use others api.can author support the plugin to use the kimi ai api.the moonshot

**Comment by logancyang** [comment]
Do they have an API and where can I find it?

**Reply by xiaodashuaige** [reply]
I'm sorry I didn't see your reply until now, kimi ai API can be obtained at https://platform.moonshot.cn/console/account. This is their official documentation at https://platform.moonshot.cn/docs/intro
